build-dev.xml

```diff
@@ -580,38 +580,11 @@
         </exec>
     </target>
 
-    <target name="test-db-fixtures-demo-multidomain" description="Loads multidomain demo data fixtures excluding first domain into test DB.">
-        <if>
-            <istrue value="${is-multidomain}" />
-            <then>
-                <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
-                    <arg value="${path.bin-console}" />
-                    <arg value="--env=test" />
-                    <arg value="shopsys:fixtures:load" />
-                    <arg value="--fixtures=${path.framework}/src/DataFixtures/DemoMultidomain" />
-                    <arg value="--fixtures=${path.src}/Shopsys/ShopBundle/DataFixtures/DemoMultidomain" />
```

Comment:
hi, what do we want to do with upgrade of datafixtures

Reply:
oh yes :man_facepalming: Boris, thanks for noticing, we need to fix that :+1:

Reply:
Data fixtures resolved in f86cf2dc2a028dbf2f5fa0e81ccec0954958c499

Reply:
OK, i see still 2 folders in Datafixtures but maybe it is ok for different project with different implementation as project-base